content/data-sourcing.md

```diff
@@ -11,33 +11,33 @@ nav_order: 4
 
 The major challenge remaining for ASmap is to source and aggregate data about which AS owns which IP address range. Several entities, public and private, make BGP announcement data available. The following data sources represent the current state of research, and all remain incomplete or problematic in some way.
 
-### RIPE RIS
+### RPKI
 
-RIPE RIS has been used as the source for all ASMaps so far, but the project describes itself the data as "useful for looking at the state of the BGP Internet, debugging/post-mortems of events in BGP, and tracking of long-term trends in BGP" [RIS docs](https://ris.ripe.net/docs/mrt/). As such, data is collected for research purposes to observe and examine malicious behavior after the fact and learn from it. This means RIS would go out of its way to include data such as BGP hijacks and fat-finger type of leaks. There is no filtering or clean-up (after a leak was identified) of this data. The RRC's location and functionality are also publicly known so sending malicious announcements with the goal of getting them into RIS is comparatively easy.
+We can download [RPKI](http://localhost:4000/glossary/#rpki-resource-public-key-infrastructure) [ROAs](http://localhost:4000/glossary/#roa-route-origin-authorization) and build a prefix to AS mapping from it that is validated by the trust chain from one of the RIRs. This gives us a much higher quality of data.
 
-This means the RIS data is not used anywhere else to inform routing decisions directly, to my knowledge we would be the first to use the data in this way. RIS is also a more centralized solution than the alternatives, though RIPE remains an important data source and thus would not be a sound data source to build on.
+Open-source validator software that does most of the heavy lifting for us is available. The most likely candidates to be used are RPKI-client and Routinator as shown by analysis of a variety of factors recently (see [Tools](http://localhost:4000/tools-and-resources/)).
 
-I believe we fare best if we only use RIPE RIS and comparable data sources as the input of last resort, when certain mappings are not available anywhere else.
+The downside of RPKI is that its data is not complete since RPKI is not as widely deployed as we would like. If RPKI is not deployed by an AS their prefixes cannot be validated and will not be present in an RPKI repository. In a test in mid 2023, I found about 60% of the Bitcoin network can be validated. This is a good base but not enough.
```

Comment:
Should remove "I" IMO because it's not clear who this is anymore if multiple people edit this page and it's not a direct quote. But  we can discuss such style stuff at a later point too.

```suggestion
The downside of RPKI is that its data is not complete since RPKI is not as widely deployed as we would like. If RPKI is not deployed by an AS their prefixes cannot be validated and will not be present in an RPKI repository. In a test in mid 2023, fjahr found about 60% of the Bitcoin network can be validated. This is a good base but not enough.
```

Reply:
thanks yea I missed this one. Went for a neutral "we" instead.